src/api-serverless/openapi.yaml

```diff
@@ -789,5 +907,221 @@ components:
             items:
                 $ref: '#/components/schemas/DropComment'
 
+    CreateNewProfileProxy:
+      type: object
+      required:
+        - target_id
+      properties:
+        target_id:
+          type: string
+
+    ProfileProxy:
+      type: object
+      required:
+        - id
+        - target_id
+        - created_at
+        - created_by_id
+      properties:
+        id:
+          type: string
+        target_id:
+          type: string
+        created_at:
+          type: number
+        created_by_id:
+          type: string
+
+    CreateNewProfileProxyAllocateRepAction:
+      type: object
+      required:
+        - action_type
+        - start_time
+        - end_time
+        - credit
+      properties:
+        action_type:
+          $ref: '#/components/schemas/CreateNewProfileProxyActionType'
+        start_time:
+          type: number
+          format: int64
+        end_time:
+          anyOf:
+            - type: 'null'
+            - type: number
+              format: int64
+        credit:
+          type: number
+          format: int64
+        group_id:
+          anyOf:
+            - type: 'null'
+            - type: string
+        category:
+          anyOf:
+            - type: 'null'
+            - type: string
+
+    CreateNewProfileProxyAllocateCicAction:
+      type: object
+      required:
+        - action_type
+        - start_time
+        - end_time
+        - credit
+        - group_id
+      properties:
+        action_type:
+          $ref: '#/components/schemas/CreateNewProfileProxyActionType'
+        start_time:
+          type: number
+          format: int64
+        end_time:
+          anyOf:
+            - type: 'null'
+            - type: number
+              format: int64
+        credit:
+          type: number
+          format: int64
+        group_id:
+          anyOf:
+            - type: 'null'
+            - type: string
+
+    CreateNewProfileProxyCreateWaveAction:
+      type: object
+      required:
+        - action_type
+        - start_time
+        - end_time
+      properties:
+        action_type:
+          $ref: '#/components/schemas/CreateNewProfileProxyActionType'
+        start_time:
+          type: number
+          format: int64
+        end_time:
+          anyOf:
+            - type: 'null'
+            - type: number
+              format: int64
+
+    CreateNewProfileProxyReadWaveAction:
+      type: object
+      required:
+        - action_type
+        - start_time
+        - end_time
+      properties:
+        action_type:
+          $ref: '#/components/schemas/CreateNewProfileProxyActionType'
+        start_time:
+          type: number
+          format: int64
+        end_time:
+          anyOf:
+            - type: 'null'
+            - type: number
+              format: int64
+
+    CreateNewProfileProxyCreateDropToWaveAction:
```

Comment:
Maybe `CreateNewProfileProxyCreateWaveParticipationDropAction` would be less ambiguous